Backport upstream change to use irqbalance_banned_cpus_(clear|setup) functions.

See: https://github.com/redhat-performance/tuned/commit/4a8e64994eb978ada0c7e23702e96c82352cc222

diff -Nura a/usr/lib/tuned/cpu-partitioning/script.sh b/usr/lib/tuned/cpu-partitioning/script.sh
--- a/usr/lib/tuned/cpu-partitioning/script.sh
+++ b/usr/lib/tuned/cpu-partitioning/script.sh
@@ -39,8 +39,7 @@
     mkdir -p "${TUNED_tmpdir}/usr/lib/dracut/hooks/pre-udev"
     cp /etc/systemd/system.conf "${TUNED_tmpdir}/etc/systemd/"
     cp 00-tuned-pre-udev.sh "${TUNED_tmpdir}/usr/lib/dracut/hooks/pre-udev/"
-    sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance
-    echo "IRQBALANCE_BANNED_CPUS=$TUNED_isolated_cpumask" >>/etc/sysconfig/irqbalance
+    irqbalance_banned_cpus_setup "$TUNED_isolated_cpumask"
     setup_kvm_mod_low_latency
     disable_ksm
 
@@ -52,7 +51,7 @@
 stop() {
     if [ "$1" = "full_rollback" ]
     then
-        sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance
+        irqbalance_banned_cpus_clear
         teardown_kvm_mod_low_latency
         enable_ksm
     fi
